src/sections/Meshmap/Meshmap-collaborate/index.js

```diff
@@ -6,12 +6,14 @@ import CollaborationFeatureWork from "./collaboration-feature-work";
 import DesignDefault from "../../Home/MeshmapDesignHighlight";
 import SignupForm from "./../signup-form";
 import Reviews from "../../Pricing/review-slider";
+import CollaborationFeatureCreate from "./collaboration-feature-terms";
 
 const MeshmapDesign = ({ theme }) => {
   return (
     <MeshmapCollaborateWrapper>
       <MeshmapCollaborateBanner theme={theme} />
       <CollaborationFeatureTeam theme={theme} />
+      <CollaborationFeatureCreate/>
```

Comment:
You can comment this line out so we can merge this PR.

Reply:
ok great